examples/idea-examples/titanic/src/main/kotlin/org/jetbrains/kotlinx/dataframe/examples/titanic/ml/titanic.kt

```diff
@@ -64,18 +79,23 @@ fun main() {
     }
 }
 
-fun <T> DataFrame<T>.toTrainTest(trainRatio: Double, yColumn: ColumnSelector<T, Number>): Pair<OnHeapDataset, OnHeapDataset> =
-    toOnHeapDataset(yColumn).split(trainRatio)
+fun <T> DataFrame<T>.toTrainTest(
```

Comment:
let's agree to create separate commits for this kind of changes? it will be easier to review without taking time to make sure semantics haven't changed, because now it's a bit painful

Reply:
I want to, but even having a file opened, after I press run, then IntelliJ lints it XD Then it feels wrong to undo it again. It's good practice for code I write myself.
I'll go look for that setting next. I understand it's difficult to check it like this, so I'll try. 
However, the entire project could do with a good linting, cause in almost every file there are long lines, mixed named and positional arguments, missing spaces before brackets etc. etc. Will get to that one day :)